src/index.html

```diff
@@ -1,13 +1,377 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MYBIKE</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
+    <link rel="icon" href="./images/favicon.svg">
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="page__header header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a
+              href="/"
```

Comment:
Logo-link doesn't work. Fix it
```suggestion
              href="#"
```